systems/plants/shapes/Geometry.h

```diff
@@ -0,0 +1,91 @@
+#ifndef __DrakeShapesGeometry_H__
+#define __DrakeShapesGeometry_H__
+
+#include <string>
+
+#include <Eigen/Dense>
+#include <Eigen/StdVector>
+
+#include "drakeShapesMacros.h"
+
+namespace DrakeShapes
+{
+  enum DLLEXPORT_drakeShapes Shape {
```

Comment:
isn't it safer to have these set explicitly, since you are relying on the numerical value of the enum in the code?  

(e.g. UNKNOWN=0, BOX=1, ... , CAPSULE=6)